src/system-health/scripts/healthd

```diff
@@ -68,19 +68,22 @@ class HealthDaemon(DaemonBase):
         """
         self.log_notice("Starting up...")
 
-        import sonic_platform.platform
-        chassis = sonic_platform.platform.Platform().get_chassis()
-        manager = HealthCheckerManager()
-        if not manager.config.config_file_exists():
-            self.log_warning("System health configuration file not found, exit...")
-            return
-        while 1:
-            state, stat = manager.check(chassis)
-            if state == HealthCheckerManager.STATE_RUNNING:
-                self._process_stat(chassis, manager.config, stat)
-
-            if self.stop_event.wait(manager.config.interval):
-                break
+        try:
+            import sonic_platform.platform
+            chassis = sonic_platform.platform.Platform().get_chassis()
+            manager = HealthCheckerManager()
+            if not manager.config.config_file_exists():
+                self.log_warning("System health configuration file not found, exit...")
+                return
+            while 1:
+                state, stat = manager.check(chassis)
+                if state == HealthCheckerManager.STATE_RUNNING:
+                    self._process_stat(chassis, manager.config, stat)
+
+                if self.stop_event.wait(manager.config.interval):
+                    break
+        except ImportError as _:
```

Comment:
Since we're not using the exception, suggest simplifying to:

```python
except ImportError:
```

You can also make the same change on line 113 of determine-reboot-cause.

Reply:
Thanks for catching this. Fixed.